docs/integrations/sources/surveymonkey.md

```diff
@@ -1,53 +1,53 @@
-# SurveyMonkey
+# Set up the SurveyMonkey Source Connector
 
-This page guides you through the process of setting up the SurveyMonkey source connector.
-
-:::note
-
-OAuth for Survey Monkey is officially supported only for the US. We are testing how to enable it in the EU at the moment. If you run into any issues, please [reach out to us](mailto:product@airbyte.io) so we can promptly assist you.
-
-:::
 
 <!-- env:oss -->
 
 ## Prerequisites
 
+**SurveyMonkey:**
+- Registered Private application [here](https://developer.surveymonkey.com/apps/)
+- SurveyMonkey authorization credentials
+- SurveyMonkey authorization method (oauth2.0)
+
 **For Airbyte Open Source:**
 
 - Access Token
+- Start date for the Source Connector (in UTC)
 <!-- /env:oss -->
 
-## Setup guide
+## Step 1: Set up SurveyMonkey
 
-### Step 1: Set up SurveyMonkey
-
-Please read this [docs](https://developer.surveymonkey.com/api/v3/#getting-started). Register your application [here](https://developer.surveymonkey.com/apps/) Then go to Settings and copy your access token
-
-### Step 2: Set up the source connector in Airbyte
+1. Log in to SurveyMonkey.
+2. Then go to Settings and copy the Private app access token.
```

Comment:
```suggestion
2. From [your list of Apps](https://developer.surveymonkey.com/apps/) on SurveyMonkey, navigate to Settings for your Private app, copy the Access Token.
```

Reply:
Important because without this clarification, users may log into SurveyMonkey and discover there is nothing labeled "Settings"...which they will only find in association with the developer.surveymonkey.com/apps page.